test/test_other.py

```diff
@@ -8650,8 +8650,8 @@ def get_addr(address):
     # as described by the DWARF.
     # TODO: consider also checking the function names once the output format
     # stabilizes more
-    self.assertRegex(get_addr('0x124').replace('\n', ''),
-                     'test_dwarf.c:15:3.*test_dwarf.c:20:3')
+    self.assertRegex(get_addr('0x118').replace('\n', ''),
```

Comment:
There was no change to the codegen here, these just changed because code was deleted from the test.